src/test/java/core/basesyntax/PurchaseOperationTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.operation.BalanceOperationHandler;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.PurchaseOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class PurchaseOperationTest {
+    @Test
+    void purchaseFruits_Ok() {
+        OperationHandler balanceOperationHandler = new BalanceOperationHandler();
+        balanceOperationHandler.processOperation("Banana", 10);
+        balanceOperationHandler.processOperation("Apple", 20);
+        balanceOperationHandler.processOperation("Grape", 3);

```

Comment:
it is better not to use other operators here. You can directly fill the database with data as test data